format/spec.md

```diff
@@ -1025,28 +1033,29 @@ Values should be stored in Parquet using the types and logical type annotations
+| **`date`**           | `int32`                                                            | `DATE`                                | Stores days from 1970-01-01.                                   |
+| **`time`**           | `int64`                                                            | `TIME_MICROS` with `adjustToUtc=false` | Stores microseconds from midnight.                             |
+| **`timestamp`**      | `int64`                                                            | `TIMESTAMP_MICROS` with `adjustToUtc=false` | Stores microseconds from 1970-01-01 00:00:00.000000.           |
+| **`timestamptz`**    | `int64`                                                            | `TIMESTAMP_MICROS` with `adjustToUtc=true` | Stores microseconds from 1970-01-01 00:00:00.000000 UTC.       |
+| **`timestamp_ns`**   | `int64`                                                            | `TIMESTAMP_NANOS` with `adjustToUtc=false` | Stores nanoseconds from 1970-01-01 00:00:00.000000000.         |
+| **`timestamptz_ns`** | `int64`                                                            | `TIMESTAMP_NANOS` with `adjustToUtc=true` | Stores nanoseconds from 1970-01-01 00:00:00.000000000 UTC.     |
+| **`string`**         | `binary`                                                           | `UTF8`                                | Encoding must be UTF-8.                                        |
+| **`uuid`**           | `fixed_len_byte_array[16]`                                         | `UUID`                                |                                                                |
+| **`fixed(L)`**       | `fixed_len_byte_array[L]`                                          |                                       |                                                                |
+| **`binary`**         | `binary`                                                           |                                       |                                                                |
+| **`struct`**         | `group`                                                            |                                       |                                                                |
+| **`list`**           | `3-level list`                                                     | `LIST`                                | See Parquet docs for 3-level representation.                   |
+| **`map`**            | `3-level map`                                                      | `MAP`                                 | See Parquet docs for 3-level representation.                   |
+| **`variant`**        | `group` with `Data` and `Metadata` fields of `binary` type         |                                       | See Parquet docs for Variant encoding.                         |
```

Comment:
This is inaccurate: the field names are case sensitive and one is named `value` not `data`.

I think this should link to the Parquet docs, one for shredded and one for non-shredded values. The physical type should be:
> `group` of `metadata` and `value` fields

The logical type should be `VARIANT` (See https://github.com/apache/parquet-format/pull/460).